### DISCUSSION TITLE
Add trace automatically when send_log_calls_to_ray = true

### DISCUSSION THREAD

**Discussion by marnickmenting** [spatie/laravel-ray#158] [Ideas]
Right now Ray logs the errors from Laravel in Ray. That is great. But it would be more helpful to directly see the trace. Now it always references the Hander.php class, which is not very useful, so I need to find the error in the laravel.log file anyway.

I know you can do ray()->trace() but that is intentional, I would like to have it by default on all errors.

![Schermafbeelding 2021-02-11 om 11 35 18](https://user-images.githubusercontent.com/10233625/107625904-4cbac900-6c5d-11eb-9f57-451444f837e3.png)

**Comment by freekmurze** [comment]
We're currently working on displaying exceptions much nicer. We'll probably add this next week.
